### PR TITLE
CLDR-14297 update ICU4J libs version in tools/pom.xml

### PR DESCRIPTION
**Pull request by pedberg-icu**
##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-14297
- [x] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

File: tools/pom.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
                     Note: see https://github.com/unicode-org/icu/packages/411079/versions for the
                     icu4j.version tag to use
                 -->
-		<icu4j.version>68.1-cldr-2020-10-21</icu4j.version>
+		<icu4j.version>68.1-release-68-1</icu4j.version>
```
Comment (srl295) [inline tools/pom.xml:22]:
```suggestion
		<icu4j.version>68.1-release-68-1</icu4j.version>
```

See the URL on line 19 - you can see the exact tag names. If you click one such as https://github.com/unicode-org/icu/packages/411079?version=68.1-release-68-1 you can see them in detail: `<version>68.1-release-68-1</version>`

**Review by srl295 — CHANGES_REQUESTED** [review]

**Review by srl295 — DISMISSED** [review]
I think that the asset may just not have built when this ran.

**Comment by srl295** [comment]
I think I know what's going on here.

Since the PR is in _your fork_, Github actions does NOT supply the secrets that it would have if it was in the head fork.
Normally this asset would be cached, but since this is the first time it's seen 68.1 release, it doesn't work.

Update: no, that didn't fix it. hmm

**Comment by srl295** [comment]
@pedberg-icu you might need to rebase on top of https://github.com/unicode-org/cldr/pull/849 to get this to work

**Comment by srl295** [comment]
@pedberg-icu rebased branch in https://github.com/unicode-org/cldr/tree/CLDR-14297-update-ICU-libs-via-pom

**Comment by pedberg-icu** [comment]
@srl295 , looks like you did the rebase but this is stil faialing, e.g. the cldr-mvn / build (push) log shows:
```
Error:  Failed to execute goal on project cldr-code: Could not resolve dependencies for project org.unicode.cldr:cldr-code:jar:39.0-SNAPSHOT: Failed to collect dependencies at com.ibm.icu:icu4j-for-cldr:jar:68.1-release-68-1: Failed to read artifact descriptor for com.ibm.icu:icu4j-for-cldr:jar:68.1-release-68-1: Could not transfer artifact com.ibm.icu:icu4j-for-cldr:pom:68.1-release-68-1 from/to github (https://maven.pkg.github.com/unicode-org/cldr): Transfer failed for https://maven.pkg.github.com/unicode-org/cldr/com/ibm/icu/icu4j-for-cldr/68.1-release-68-1/icu4j-for-cldr-68.1-release-68-1.pom 400 Bad Request -> [Help 1]
```

**Comment by srl295** [comment]
> @srl295 , looks like you did the rebase but this is stil failing, e.g. the cldr-mvn / build (push) log shows:
> 
> ```
> 
> Error:  Failed to execute goal on project cldr-code: Could not resolve dependencies for project org.unicode.cldr:cldr-code:jar:39.0-SNAPSHOT: Failed to collect dependencies at com.ibm.icu:icu4j-for-cldr:jar:68.1-release-68-1: Failed to read artifact descriptor for com.ibm.icu:icu4j-for-cldr:jar:68.1-release-68-1: Could not transfer artifact com.ibm.icu:icu4j-for-cldr:pom:68.1-release-68-1 from/to github (https://maven.pkg.github.com/unicode-org/cldr): Transfer failed for https://maven.pkg.github.com/unicode-org/cldr/com/ibm/icu/icu4j-for-cldr/68.1-release-68-1/icu4j-for-cldr-68.1-release-68-1.pom 400 Bad Request -> [Help 1]
> 
> ```

I didn't rebase your branch. But on my own branch it worked. So if you rebase it should be ok.

**Review by srl295 — APPROVED** [review]

**Comment by pedberg-icu** [comment]
Great, thanks, the rebasing to pick up #849 does seems to have fixed things.

**Pull request merged**